### PR TITLE
Broader localonly

### PR DESCRIPTION
**Pull request by brijeshnec**

### CODE REVIEW

File: HistoryQueryManager/src/main/java/eu/neclab/ngsildbroker/historyquerymanager/controller/HistoryController.java [hunk 1]
```diff
@@ -105,7 +105,7 @@ public Uni<RestResponse<Object>> queryTemporalEntities(HttpServerRequest request
 			return Uni.createFrom()
 					.item(HttpUtils.handleControllerExceptions(new ResponseException(ErrorType.TooManyResults)));
 		}
-		if (typeQuery == null && attrs == null && geometry == null && q == null) {
+		if (!localOnly && typeQuery == null && attrs == null && geometry == null && q == null) {
```
Comment (amitrnec) [inline HistoryQueryManager/src/main/java/eu/neclab/ngsildbroker/historyquerymanager/controller/HistoryController.java:108]: Validate this condition in all cases

File: QueryManager/src/main/java/eu/neclab/ngsildbroker/queryhandler/controller/QueryController.java [hunk 2]
```diff
@@ -216,7 +216,7 @@ public Uni<RestResponse<Object>> query(HttpServerRequest request, @QueryParam("i
 			return Uni.createFrom()
 					.item(HttpUtils.handleControllerExceptions(new ResponseException(ErrorType.TooManyResults)));
 		}
-		if (id == null && typeQuery == null && attrs == null && geometry == null && q == null && pick == null) {
+		if (!localOnly && id == null && typeQuery == null && attrs == null && geometry == null && q == null && pick == null) {
```
Comment (amitrnec) [inline QueryManager/src/main/java/eu/neclab/ngsildbroker/queryhandler/controller/QueryController.java:220]: Validate pick here

File: HistoryQueryManager/src/main/java/eu/neclab/ngsildbroker/historyquerymanager/controller/HistoryController.java (hunk 1 above)
Comment (brijeshnec) [reply HistoryQueryManager/src/main/java/eu/neclab/ngsildbroker/historyquerymanager/controller/HistoryController.java:108]: done

File: QueryManager/src/main/java/eu/neclab/ngsildbroker/queryhandler/controller/QueryController.java (hunk 2 above)
Comment (brijeshnec) [reply QueryManager/src/main/java/eu/neclab/ngsildbroker/queryhandler/controller/QueryController.java:220]: done

**Review by ScorpioBroker — CHANGES_REQUESTED** [review]
please update update test to be mergeable

**Review by ScorpioBroker — APPROVED** [review]

**Pull request merged**